plugin/customcertificate/src/print_certificate.php

```diff
@@ -485,25 +491,61 @@ class = "caraA"
         'bottom' => 0,
     ];
     $pdf = new PDF($params['format'], $params['orientation'], $params);
-    if (count($htmlList) == 1) {
-        $pdf->content_to_pdf($content, '', $fileName, null, 'D', false, null, false, false, false);
-        exit;
-    } else {
-        $filePath = $archivePath.$fileName.'.pdf';
-        $pdf->content_to_pdf($content, '', $fileName, null, 'F', true, $filePath, false, false, false);
-        $fileList[] = $filePath;
+
+    $contentAllCertificate = '';
+    foreach ($htmlList as $fileName => $content) {
+        $contentAllCertificate .= $content;
     }
-}
 
-if (!empty($fileList)) {
-    $zipFile = $archivePath.'certificates_'.api_get_unique_id().'.zip';
-    $zipFolder = new PclZip($zipFile);
-    foreach ($fileList as $file) {
-        $zipFolder->add($file, PCLZIP_OPT_REMOVE_ALL_PATH);
+    if (!empty($contentAllCertificate)) {
+        $certificateContent = '<html>';
+        $certificateContent .= '
+        <link rel="stylesheet"
+            type="text/css"
+            href="'.api_get_path(WEB_PLUGIN_PATH).'customcertificate/resources/css/certificate.css">';
+        $certificateContent .= '
+        <link rel="stylesheet"
+            type="text/css"
+            href="'.api_get_path(WEB_CSS_PATH).'document.css">';
+        $certificateContent .= '<body>';
+        $certificateContent .= $contentAllCertificate;
+        $certificateContent .= '</body></html>';
+
+        $pdf->content_to_pdf($certificateContent, '', 'certificate'.date("Y_m_d_His"), null, 'D', false, null, false, false, false);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line exceeds maximum limit of 120 characters; contains 132 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5687817)